code/modules/clothing/head/helmet.dm

```diff
@@ -84,8 +84,11 @@
 	tint = 2
 
 /obj/item/clothing/head/helmet/blueshirt
+	name = "blue helmet"
+	desc = "A reliable, blue tinted helmet, you can't but help to think of beer."

```

Comment:
"can't but help"?

I feel like this could be worded better. Is it a reference to a specific quote?

Reply:
I'll word it better anyway